webpack/config.dev.js

```diff
@@ -122,17 +122,8 @@ module.exports = [{
         loader: 'babel-loader',
         query: {
           presets: [
-            'react',
-            'env',
-            'stage-0',
-          ],
-          plugins: [
-            [
-              'babel-plugin-webpack-loaders', {
-                'config': path.resolve(__dirname, './config.babel.js'),
-                'verbose': false
-              }
-            ]
+            '@babel/preset-react',
```

Comment:
Is it possible for the webpack babel-loader plugin to read from `.babelrc` so that `@babel/preset-react` and `@babel/preset-env` don't need to be duplicated in the webpack configs too?

The [documentation](https://github.com/babel/babel-loader) isn't very clear about this.